syntax/abstract.mjs

```diff
@@ -36,9 +36,18 @@ export function list_into(Type) {
                     entries
                         .reduce(join_adjacent(FTL.Junk), [])
                         .filter(remove_blank_lines)));
+        case FTL.SelectExpression:
+            return ([selector, variants]) =>
+                variants.some(variant =>
+                    variant.value instanceof FTL.VariantList)
+                    ? never("")
```

Comment:
Please add an actual error message here? Maybe `Select expressions can only have patterns or select expressions as values` ?

Reply:
Sorry, I missed this after I resolved a conflict with #128.